kolibri/core/tasks/iceqube/utils.py

```diff
@@ -0,0 +1,117 @@
+import importlib
+import logging
+import time
+import uuid
+
+from kolibri.core.tasks.iceqube import compat
+
+
+# An object on which to store data about the current job
+# So far the only use is to track the job_id, but other metadata
+# could be added.
+current_job_tracker = compat.local()
+
+
+def get_current_job():
+    return getattr(current_job_tracker, "job", None)
+
+
+def stringify_func(func):
+    assert callable(
+        func
+    ), "function {} passed to stringify_func isn't a function!".format(func)
+
+    fqn = "{module}.{funcname}".format(module=func.__module__, funcname=func.__name__)
+    return fqn
+
+
+def import_stringified_func(funcstring):
+    """
+    Import a string that represents a module and function, e.g. {module}.{funcname}.
+
+    Given a function f, import_stringified_func(stringify_func(f)) will return the same function.
+    :param funcstring: String to try to import
+    :return: callable
+    """
+    assert isinstance(funcstring, str)
```

Comment:
Would be [slightly preferable](https://stackoverflow.com/a/20076295) to `raise` a custom error rather than using `assert`

Reply:
Yes - for the stringify_func method above also.